src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/dialogEditBlacklist.xml

```diff
@@ -32,7 +32,6 @@
     <field>
         <id>blacklist.filter</id>
         <label>categories (if available)</label>
-        <type>text</type>
```

Comment:
maybe we should break this out and merge it immediately?

Reply:
It is not important at the moment - it is overwritten in the next line but it should be removed in the future as it might cause problems in the future (if for some reasons the other element is used).

Reply:
merge glitch